src/lib/Scenes/Home/index.tsx

```diff
@@ -97,4 +120,24 @@ export default class Home extends React.Component<Props, State> {
   openLink() {
     SwitchBoard.presentNavigationViewController(this, Router.ConsignmentsStartSubmission)
   }
+
+  setSelectedTab(selectedTab) {
+    this.setState({ selectedTab: selectedTab.i }, () => this.fireHomeScreenViewAnalytics())
+  }
+
+  fireHomeScreenViewAnalytics() {
+    let screenType
+
+    if (this.state.selectedTab === 0) {
+      screenType = Schema.PageNames.HomeArtistsWorksForYou
+    } else if (this.state.selectedTab === 1) {
```

Comment:
Maybe define constants at the top of the file for these magic numbers?

Reply:
done